README.md

```diff
@@ -1,5 +1,67 @@
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # curriculum-cr-training-contact-form
 
-This repo host an initial code for the tasks performed during the Code Reviewers training.
+> In this project I have made the contact form that is based on this [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)..
+
+![screenshot](contact.png)
+
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo
+
+[Live Demo Link](https://raw.githack.com/Dipeshtwis/curriculum-cr-training-contact-form/feature/contact-form/index.html)
```

Comment:
**Required**
- [ ] You might want to push your changes to production, as the current live demo does not implement your recent changes.